src/Bindings/LuaJson.cpp

```diff
@@ -134,7 +134,7 @@ static Json::Value JsonSerializeTable(cLuaState & a_LuaState)
 	{
 		if (lua_type(a_LuaState, -2) == LUA_TNUMBER)
 		{
-			int idx;
+			int idx = 0;
 			a_LuaState.GetStackValue(-2, idx);
```

Comment:
It can fail. Lua numbers are doubles, and this is an int. So if I write plugin code to pass 1e40 to the function, its a number but its too big to fit in an int so get stack values should fail.